### PR TITLE
Multiple values fix

### PR DESCRIPTION
**Pull request by kingsleyndiewo**
Update to combine_field_values function

Initial function assumed fields would be JSON which brought errors when they weren't. Current version fetches any previously stored values and concatenates to store as CSV

### CODE REVIEW

**Comment by moshthepitt** [comment]
@kingsleyndiewo could you describe - in a commit message or this PR description - the way the previous implementation failed and how this fixes things?

**Comment by kingsleyndiewo** [comment]
> @kingsleyndiewo could you describe - in a commit message or this PR description - the way the previous implementation failed and how this fixes things?

Updated description.

File: deploy/add_combine_field_values_function.psql [hunk 1]
```diff
@@ -8,8 +8,23 @@ BEGIN;
 -- :"schema" variable - and default to public.
```
Comment (moshthepitt) [inline deploy/add_combine_field_values_function.psql:8]: Could we add some tests for the functions in this file using sample data from the live database?  Here's an example of similar tests we've done before: https://github.com/opensrp/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/5-IRS/2-Zambia-2019/verify/process_structure_geo_hierarchy_queue.psql#L161

**Comment by kingsleyndiewo** [comment]
Function combines values into CSV. Works as expected.
Verify fixed.

File: sqitch.conf [hunk 2]
```diff
@@ -5,10 +5,12 @@
 	# top_dir = .
 [engine "pg"]
 	target = sqitch_test
-	registry = sqitch
+	registry = sqitch_afyatek
```
Comment (moshthepitt) [inline sqitch.conf:8]: Why is this change necessary?

Comment (kingsleyndiewo) [reply sqitch.conf:8]: I did this a while back when I was deploying to the stage schema on the production DB. The sqitch stuff on the production DB is in sqitch_afyatek, and not sqitch.

File: deploy/add_combine_field_values_function.psql (hunk 1 above)
Comment (kingsleyndiewo) [reply deploy/add_combine_field_values_function.psql:7]: Tests added [here](verify/store_all_for_multiple_choice_fields.psql).

**Review by moshthepitt — APPROVED** [review]
